apps/ehr/tests/component/AddPatientValidation.test.tsx

```diff
@@ -0,0 +1,327 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { BrowserRouter, useNavigate } from 'react-router-dom';
+import { beforeEach, describe, expect, it, vi } from 'vitest';
+import { dataTestIds } from '../../src/constants/data-test-ids';
+import AddPatient from '../../src/pages/AddPatient';
+
+vi.mock('react-router-dom', async () => {
```

Comment:
can we mock react router and other dependencies in a vitest config file somewhere? so we won't need to write this in every test file and add places for bugs to spawn